templates/recording-deployment.yaml

```diff
@@ -77,13 +77,15 @@ spec:
         volumeMounts:
         - mountPath: "/data"
           name: data-recordings
+        - mountPath: "/tmp"
+          name: tmp
```

Comment:
Chromium wants to store the "user profile" on disk somewhere, by default in `/tmp`, so we just map this to memory.